pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.folio</groupId>
   <artifactId>mod-oai-pmh</artifactId>
-  <version>3.0.4</version>
+  <version>3.2.0-SNAPSHOT</version>
```

Comment:
If you're releasing off this branch it should be 3.1.0

Reply:
My guess is that you already created the tag - unfortunately, that means if you need to make any adjustments - in reaction to PR comments for instance, you'll need to create a new tag and bump the version again.  The CI system doesn't handle moving the tag to a new commit very well.

Reply:
You are right, yes, it was by mistake that we are not goint to merge it to mainline but to long-live branch b3.0. I have fixed it. Thank you.